components/user-mgt/org.wso2.carbon.user.mgt.ui/src/main/resources/web/role/edit-users.jsp

```diff
@@ -404,40 +404,42 @@
                                                             disPlayName = userName;
                                                         }
                                                         if (users[i].getItemName()
-                                                                    .equals(CarbonConstants.REGISTRY_ANONNYMOUS_USERNAME)) {
+                                                                .equals(CarbonConstants.REGISTRY_ANONNYMOUS_USERNAME)) {
                                                             continue;
                                                         } else if (readOnlyRole && !users[i].getEditable()) {
                                                             doEdit = "disabled=\"disabled\"";
                                                         } else if (session.getAttribute("checkedUsersMap") != null &&
-                                                                   ((Map<String, Boolean>) session
-                                                                           .getAttribute("checkedUsersMap"))
-                                                                           .get(users[i].getItemName()) != null &&
-                                                                   ((Map<String, Boolean>) session
-                                                                           .getAttribute("checkedUsersMap"))
-                                                                           .get(users[i].getItemName()) == true) {
+                                                                ((Map<String, Boolean>) session
+                                                                        .getAttribute("checkedUsersMap"))
+                                                                        .get(users[i].getItemName()) != null &&
+                                                                ((Map<String, Boolean>) session
+                                                                        .getAttribute("checkedUsersMap"))
+                                                                        .get(users[i].getItemName()) == true) {
```

Comment:
Fixed with #915 
Since map is in Map<String, Boolean>; the return value is a Boolean. We cannot use  "
Boolean.parseBoolean(String)". Hence "Boolean.TRUE.equals(Object)" will do the needful and handle null check for the value as well.